### PR TITLE
Show the scheduled time if applicable. Closes #799

### LINKED ISSUE #799: Scheduled maintenance emails don't contain the time at which the maintenance is to occur
I've just set a couple of test scheduled maintenance windows and chosen to alert my subscribers to the event.

The emails came through, but it doesn't include the time at which the event is scheduled to happen.

### PR DESCRIPTION
**Pull request by jbrooksuk**
Closes #799

### CODE REVIEW

**Comment by jbrooksuk** [comment]
@joecohens can you test this as well please.

File: app/Presenters/IncidentPresenter.php [hunk 1]
```diff
@@ -160,7 +160,7 @@ public function icon()
     public function toArray()
     {
         return array_merge($this->wrappedObject->toArray(), [
-            'scheduled_at' => $this->created_at(),
+            'scheduled_at' => $this->scheduled_at(),
```
Comment (jbrooksuk) [inline app/Presenters/IncidentPresenter.php]: @joecohens whoops, haha!

Comment (joecohens) [inline app/Presenters/IncidentPresenter.php]: :stuck_out_tongue:

File: app/Http/Controllers/Admin/ScheduleController.php [hunk 2]
```diff
@@ -109,7 +109,7 @@ public function addScheduleAction()
         $scheduleData['status'] = 0;
 
         try {
-            Incident::create($scheduleData);
+            $incident = Incident::create($scheduleData);
```
Comment (joecohens) [inline app/Http/Controllers/Admin/ScheduleController.php]: We had this typo but now it's working awesome!

**Comment by joecohens** [comment]
:+1:

File: app/Handlers/Events/SendMaintenanceEmailNotificationHandler.php [hunk 3]
```diff
@@ -70,6 +70,7 @@ public function handle(MaintenanceHasScheduledEvent $event)
                 'status'          => $data->humanStatus,
                 'htmlContent'     => $data->formattedMessage,
                 'textContent'     => $data->message,
+                'scheduled_at'    => $data->scheduled_at_formatted,
                 'token'           => $subscriber->token,
                 'unsubscribeLink' => route('subscribe.unsubscribe', ['code' => $subscriber->verify_code]),
```
Comment (GrahamCampbell) [inline app/Handlers/Events/SendMaintenanceEmailNotificationHandler.php]: snake case needed

File: app/Handlers/Events/SendMaintenanceEmailNotificationHandler.php [hunk 4]
```diff
@@ -70,6 +70,7 @@ public function handle(MaintenanceHasScheduledEvent $event)
...
                 'appUrl'          => env('APP_URL'),
```
Comment (GrahamCampbell) [inline app/Handlers/Events/SendMaintenanceEmailNotificationHandler.php]: snake case needed

File: app/Handlers/Events/SendMaintenanceEmailNotificationHandler.php [hunk 5]
```diff
@@ -70,6 +70,7 @@ public function handle(MaintenanceHasScheduledEvent $event)
                 'status'          => $data->humanStatus,
                 'htmlContent'     => $data->formattedMessage,
```
Comment (GrahamCampbell) [inline app/Handlers/Events/SendMaintenanceEmailNotificationHandler.php]: snake case needed

File: app/Handlers/Events/SendMaintenanceEmailNotificationHandler.php [hunk 6]
```diff
@@ -70,6 +70,7 @@ public function handle(MaintenanceHasScheduledEvent $event)
...
                 'textContent'     => $data->message,
```
Comment (GrahamCampbell) [inline app/Handlers/Events/SendMaintenanceEmailNotificationHandler.php]: snake case needed

**Pull request merged**